### DISCUSSION TITLE
[Platformio] Linker, how to reserve 4 pages of 4k for EEPROM emulation

### DISCUSSION THREAD

**Discussion by hitech95** [earlephilhower/arduino-pico#1632] [Q&A]
I have an application were i have to write some data in the emulated eeprom.
I woulld implement a simple wear leveling algorythm.

To do so I need some 4k pages to rotate the writes to.
I've seen that there is a `_EEPROM_start` variable in the linker but I've not figured out how this is configured.

Any suggestion?

**Comment by earlephilhower** [comment]
You're going to need to edit the `boards.txt` file where `XXX>build.eeprom_start` is defined (you'll need to change `fs_end` too if you want to use LittleFS.

**Comment by hitech95** [comment]
At the end I'm using the little FS range for my EEPROM emulation. I've seen
that the 4096 EEPROM size is hard coded in a lot of places.

Since I don't need LittleFs I'm just using it's handy configuration. Only
"downside" I'm wasting 4k of flash.
